public/src/lobby/FileUpload.jsx

```diff
@@ -14,16 +14,23 @@ registerPlugin(FilePondPluginFileValidateSize, FilePondPluginFileValidateType);
 
 const FileUpload = () => {
   const [files, setFiles] = useState([]);
+  const [errorMsg, setErrorMsg] = useState("");
   return (
     <fieldset className='fieldset'>
-      <legend className='legend'>Upload Custom Set</legend>
+      <legend className='legend'>Upload Custom Set (Json/XML)</legend>
```

Comment:
Use capital letters? `JSON/XML`

But I think it doesn't need to be in the headline. It could be announced in the description of the box instead?